setup.py

```diff
@@ -14,7 +14,7 @@
 requirements = [
     'beautifulsoup4 ~=4.8.0',
     'celery >= 4.2.0',
-    'django >= 2.2',
+    'django ~= 2.2',
```

Comment:
The former statement installs Django 3, this library is not yet ported to Django 3.